spec/helpers/fake_gluster.rb

```diff
@@ -178,6 +235,15 @@ def add_volumes(*volumes)
     volumes.each { |volume| add_volume(volume) }
   end
 
+  def get_volume(name)
+    (volume,) = @volumes.select { |vol| vol.name == name }
```

Comment:
Bleh, I was thinking in Pythonland where `(foo,) = somelist` explodes unless the list contains exactly one element. I want to make sure there's exactly one result, so I guess I'll need to write it more verbosely.

I moved away from `Enumerable#find` because that returns the first result even if there are several matches.